sandbox.sh

```diff
@@ -189,11 +190,33 @@ start_deku_cluster() {
     deku-cli sign-block "$data_directory/$i" $HASH
   done
 
+}
+
+wait_for_servers() {
   for PID in ${SERVERS[@]}; do
     wait $PID
   done
 }
 
+assert_deku_state() {
+  contract=$(cat "$data_directory/0/tezos.json" | jq '.consensus_contract' | xargs)
+  storage=$(curl "$RPC_NODE/chains/main/blocks/head/context/contracts/$contract/storage")
```

Comment:
Shouldn't we prefer `curl --silent`?

Reply:
Should we? I think you're the superier bash scripter here. I've addressed it in the latest commit.